docker/postprocessing.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
If you're spending a lot of time managing this dockerfile, I'd suggest you use one of the fv3net-published ones instead. As it is this is duplicated work from what the ML team is already doing. These are pushed to dockerhub on every PR and tagged by fv3net git sha.

Reply:
That is a good point, did not realize fv3net has that. I was just trying to keep it as minimal as possible because this needs to be pulled from daint through sarus, and I was running into some problems with libraries when I had more requirements. I think for now we can just use this, but if it becomes a problem we should look into switching to fv3net instead of having to maintain this.